src/routes/entry-point.ts

```diff
@@ -0,0 +1,20 @@
+import Router from '@koa/router';
+import http from 'http2';
+import { Context, Middleware, Next } from 'koa';
+import { schema } from 'rdf-namespaces';
+import url from 'url';
+import Routes from './index';
+
+export default (router: Router): Middleware => (
+  async ({ request, response }: Context, next: Next): Promise<void> => {
+    response.status = http.constants.HTTP_STATUS_OK;
+    response.body = {
+      '@id': url.resolve(request.origin, router.url(Routes.EntryPoint, {})),
```

Comment:
want to test `request.origin` in practice, but seems the most flexible solution to allow simple load balancing. To be tested whether it allows reverse proxies such as gateways or CDNs using `X-Forwarded-Host`.

Reply:
Needs a setting (see https://koajs.com/#settings).